docs/r2/scalars_and_keras.ipynb

```diff
@@ -207,9 +207,14 @@
         "\n",
         "You're now ready to define, train and evaluate your model. \n",
         "\n",
-        "To log the *loss* scalar as you train, you'll create the Keras TensorBoard callback, specifying a log directory, and pass it to [Model.fit()](https://https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit).\n",
+        "To log the *loss* scalar as you train, you'll do the following:\n",
```

Comment:
I don't think "\n" is needed (here and in the following lines). There is also and extra comma.

Reply:
I've ensured it looks correct in Colab now.

Reply:
I've made sure it looks right in Colab now.